com.zsmartsystems.zigbee.console.main/src/main/java/com/zsmartsystems/zigbee/console/main/ZigBeeConsole.java

```diff
@@ -293,7 +292,7 @@ public void start() {
      * Processes text input line.
      *
      * @param inputLine the input line
-     * @param out the output stream
+     * @param out       the output stream
```

Comment:
I guess you have a different formatter that is adding some spaces after parameters - or at least not removing them?

Reply:
Hm, don't know how this happened; I just ran the formatter, and it removed the spaces. Thanks for the hint.